index.js

```diff
@@ -2017,6 +2017,24 @@
     return Foldable.methods.reduce(foldable)(f, x);
   }
 
+  //# foldMap :: (Monoid m, Foldable f) => (TypeRep m, a -> m, f a) -> m
+  //.
+  //. Deconstructs a foldable by mapping every element to a monoid and
+  //. concatenating the results.
+  //.
+  //. This function is derived from [`reduce`](#reduce), [`concat`](#concat)
+  //. and [`empty`](#empty).
+  //.
+  //. ```javascript
+  //. > foldMap(String, x => x.toString(), [1, 2, 3])
+  //. '123'
```

Comment:
I don't love this example. `x => x.toString()` could be written `String`, but then we'd have `foldMap(String, String, [1, 2, 3])` which is slightly confusing. I can't think of a good example, but this may be an improvement:

```javascript
> foldMap(String, f => f.name, [Math.sin, Math.cos, Math.tan])
'sincostan'
```